tutorials/tinystories/main.py

```diff
@@ -173,7 +172,7 @@ def run_curation_pipeline(args: argparse.Namespace, jsonl_dir: str) -> None:
         keep_extensions="jsonl",
     )
     print("Reading the data...")
-    orig_dataset = DocumentDataset.read_json(files, add_filename=True)
+    orig_dataset = DocumentDataset.read_json(files, add_filename="_filename")
```

Comment:
Is this needed if the dataset already has a "file_name" column? Are we able to write with that instead?